pkg/handlers/index/list.go

```diff
@@ -50,3 +49,37 @@ func List(c *gin.Context) {
 
 	c.JSON(http.StatusOK, items)
 }
+
+// @Id IndexNameList
+// @Summary List index Name
+// @Tags    IndexName
```

Comment:
The `tags` used to group APIs, you should refer other API how to write it. it should be `Index`